platformio.ini

```diff
@@ -202,6 +202,7 @@ lib_deps =
   #https://github.com/lorol/LITTLEFS.git
   ESPAsyncTCP @ 1.2.2
   ESPAsyncUDP
+  ESP8266PWM
```

Comment:
I think that code from "lib" is even compiled when it's not in `lib_deps` (but maybe I'm wrong...)

Reply:
It is not!  That's expressly why I put it there, to avoid having to add #ifdef guards so as to keep the source as close as possible to the upstream version.  The local `libs` folder is merely the first place in the `lib_deps` search path, so local libraries will shadow upstream ones.  It looks like it was intended as an easy place for vendoring in forked libraries.